src/TraceEvent/ETWTraceEventSource.cs

```diff
@@ -383,6 +386,23 @@ public ImageData(int ProcessID, string DllName, Address BaseAddress, int Size)
             public int Size;
         }
 
+        private IntPtr GetFunctionPointerForEtwLibrary(string procName)
+        {
+            IntPtr loadLibraryHandle = TraceEventNativeMethods.LoadLibrary(this.etwModulePath);
+            if (loadLibraryHandle == IntPtr.Zero)
+            {
+                throw new Exception($"LoadLibrary failed. Please check if the file (${this.etwModulePath}) exists, and is of the correct bitness.");
```

Comment:
Use DllNotFoundException.
Also, you have string interpolation inside string interpolation. Remove the inner $